homeassistant/components/recorder/const.py

```diff
@@ -29,7 +29,7 @@
 # in https://github.com/sqlite/sqlite/commit/efdba1a8b3c6c967e7fae9c1989c40d420ce64cc
 # We can increase this back to 1000 once most
 # have upgraded their sqlite version
-SQLITE_MAX_BIND_VARS = 998
+SQLITE_MAX_BIND_VARS = 20000
```

Comment:
```suggestion
SQLITE_MAX_BIND_VARS = 4000
```
I wouldn't got to high as we want the database to still be responsive between purge calls and the purge will block the database for a bit

Reply:
we can reduce the amount used in the attached test too --> more speedy test